packs/data/feats.db/shared-stratagem.json

```
@@ -21,7 +21,17 @@
         "prerequisites": {
             "value": []
         },
-        "rules": [],
+        "rules": [
+          {
+            "key": "Note",
+            "selector": "attack",
+            "text": "The creature you hit is @UUID[Compendium.pf2e.conditionitems.Flat-Footed] to a designated ally on the next attack the ally makes against the creature before the start of your next turn.",
+            "title": "@UUID[{item|sourceId}]",
+            "predicate": [
+              "devise-a-stratagem"
+            ]
+          }
```

```suggestion
          {
            "key": "Note",
            "selector": "strike-damage",
            "text": "The creature you hit is @UUID[Compendium.pf2e.conditionitems.Flat-Footed] to a designated ally on the next attack the ally makes against the creature before the start of your next turn.",
            "title": "{item|name}",
            "predicate": [
              "devise-a-stratagem"
            ]
          }
```
This will need localized. The selector also should not just be attack, as that would eventually fire off on spells too. Also just copy the predicate from devise a stratagem since there's technically more to when this fires off.